docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM centos:7 as builder
```

Comment:
Why not ubuntu based or something more modern? Centos has very old package versions so if someone would need additional tools in the container they will not be so happy.